inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/quantize/quantize_kernel_ref.cpp

```diff
@@ -43,21 +43,35 @@ CommonDispatchData QuantizeKernelRef::SetDefault(const quantize_params& params,
 
     auto output = params.output;
 
-    runInfo.gws0 = output.Batch().v;
-    runInfo.gws1 = params.packed_binary_output ? CeilDiv(output.Feature().v, 32) : output.Feature().v;
-    runInfo.gws2 = Align(output.X().v * output.Y().v * output.Z().v, 16);
+    if (output.GetLayout() == DataLayout::b_fs_yx_fsv16 && !params.packed_binary_output) {
+        static const size_t fsv = 16;
+        runInfo.gws0 = output.Batch().v;
+        runInfo.gws1 = Align(output.Feature().v, fsv);
+        runInfo.gws2 = output.Y().v * output.X().v * output.Z().v;
 
-    runInfo.lws0 = 1;
-    runInfo.lws1 = 1;
-    runInfo.lws2 = 16;
+        runInfo.lws0 = 1;
+        runInfo.lws1 = fsv;
```

Comment:
Just wondering: since this kernel doesn't use sub-group extension, maybe it could be beneficial to use full 32 simd? 
Have you maybe tested such scenario?

Reply:
@kdobros good point, it gives up to 1.5x in some cases. Updated subgroup size in both quantize kernels for fsv16 format